sensu-plugins-elasticsearch.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.version                = SensuPluginsElasticsearch::Version::VER_STRING
 
   s.add_runtime_dependency 'rest-client',       '1.8.0'
-  s.add_runtime_dependency 'elasticsearch',     '~> 1.0.14'
+  s.add_runtime_dependency 'elasticsearch',     '>= 1.0.10'
```

Comment:
@eheydrick I would imagine there is some incompatibilities, they are major versions which implies breaking changes. Unless we rely on pure rest or have 2 separate gems (ew) I see no way of guaranteeing and allowing it to span multiple major versions...